stateboard.init.lua

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env tarantool
+
+require('cartridge.stateboard').cfg()
```

Comment:
What about `setsearchroot` for Tarantool 1.10?

Reply:
It's only necessary for cartridge-based projects packaging, which is never used for bare cartridge. In `.rock/bin` it's already prefixed appropriately.